### DISCUSSION TITLE
Fragen zu weewx-wdc Version 2.3.0

### DISCUSSION THREAD

**Discussion by hoetzgit** [Daveiano/weewx-wdc#79] [Q&A] [answered]
@Daveiano

Hallo David,

ich bin etwas vorsichtig Issues zu generieren, weil ich eigene Fehler in meiner Konfiguration nicht ausschließen kann. daher diese Vorphase:

weewx-DWD Einbindung:

- Bei der Vorhersagetabelle habe ich offensichtlich bei lang=de ein Umlautproblem. Habe versucht, bspw. "ö" durch `&ouml;` zu ersetzen, ohne Erfolg. Auch scheinen in die `de.conf` z.B. Zeile `216` noch ein paar Zeichen zusätzlich mit reingerutscht zu sein.
- Leider sind deutsche Bezeichnungen oft sehr lang. So reicht z.B. "Niederschlagswahrscheinlichkeit" bis in den Wertebereich hinein und überschreibt diesen. Gibt es eine Möglichkeit, eine Worttrennung zu erreichen?
- Die Warnkarte "SchilderXX.jpg" ist evtl. statisch programmiert?

Benötigst Du weitere Infos, melde Dich gerne.
Siehe auch [Webseite](https://wdc.weiherhammer-wetter.de)

Grüße
Henry

**Comment by Daveiano** [comment, marked as answer]
Alles wirkliche Fehler des skins, danke fürs berichten! Issue dazu hier: #81
